autopush/endpoint.py

```diff
@@ -163,6 +170,10 @@ def _client_info(self):
             "uaid_hash": self.uaid_hash,
             "router_key": getattr(self, "router_key", ""),
             "channel_id": getattr(self, "chid", ""),
+            "audience": getattr(self, "aud", ""),
+            "submitter": getattr(self, "sub", ""),
+            "expires": getattr(self, "exp", ""),
+            "version": getattr(self, "version", ""),
```

Comment:
All these getattr calls are not cheap, we're adding more, and increasing our logging which is going to make the delivery of every message increasingly more CPU expensive. We should hook onto the initialization of the Handler class, and ensure all these default to "", so that all the getattr's can be removed.